modules/nf-core/gatk4spark/baserecalibrator/tests/main.nf.test

```diff
@@ -149,12 +143,96 @@ nextflow_process {
         }
 
         then {
+            assert process.success
+            assertAll(
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
+    test("homo sapiens - cram -stub") {
+        options "-stub"
+        when {
+            process {
+                """
+                input_cram = [ [ id:'test' ], // meta map
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram_crai'], checkIfExists: true),
+                    []
+                ]
+                fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+                fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
+                dict = file(params.test_data['homo_sapiens']['genome']['genome_dict'], checkIfExists: true)
+                sites = file(params.test_data['homo_sapiens']['illumina']['test_haplotc_cnn_vcf_gz'], checkIfExists: true)
+                sites_tbi = file(params.test_data['homo_sapiens']['illumina']['test_haplotc_cnn_vcf_gz_tbi'], checkIfExists: true)
+
+                input = [input_cram, fasta, fai, dict, sites, sites_tbi]
+                """
+            }
+        }
+
+        then {
+            assert process.success
             assertAll(
-                { assert process.success },
                 { assert snapshot(process.out).match() }
             )
         }
+    }
+
+    test("sarscov2 - bam - intervals -stub") {
+        options "-stub"
+        when {
+            process {
+                """
+                input_bam = [ [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
```

Comment:
Why are you using `params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam']` instead of 
`params.modules_testdata_base_path + 'genomics/...`

Reply:
just because I didn't it